_data/projects/traveler

```diff
@@ -0,0 +1,10 @@
+
+name: Traveler
+desc: Small Travelling app that shows the travel destinations in Sri Lanka
+site: https://github.com/SahanMonaara
+tags:
+- Android
+- java
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/SahanMonaara/Traveler/tree/master
```

Comment:
@SahanMonaara You need to create a label named up-for-grabs or any label name you would like and add link to that label here. For eg- if you are using "good first issue" as your label name then after creating new label your link should be 
"https://github.com/SahanMonaara/Traveler/labels/good%20first%20issue"